proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -83,43 +125,89 @@ message DebugBundleStatus {
   // Path in API to get the file.
   string filename = 5;
 
+  // Size of the debug bundle zip file.
+  int64 size = 6;
```

Comment:
You already used the `_bytes` suffix above:

```
controller_logs_size_limit_bytes
```

I think that's missing here. AIP-141 says:

> Quantities with a clear unit of measurement (such as bytes, miles, and so on) must include the unit of measurement as the suffix.